public/static/pathways/upenn_her2_early_stage_pathway.json

```diff
@@ -0,0 +1,589 @@
+{
+    "name": "HER2+ Early Stage Pathway",
+    "description": "Pathway for the Early Stage HER2+ Breast Cancer Pathway",
+    "library": "mCODE_Library.cql",
+    "version": 1.0,
```

Comment:
Is this just an arbitrary version number?  I didn't notice a version on any of the other pathways.

Reply:
Yes it is arbitrary. I can remove it. The reason it exists is because I started off with one of the modified HER2+ pathways which was on sharepoint not on the repo (because it wasn't being used yet). To distinguish between the original version and the new version I added the version tag.

Reply:
Yes it is arbitrary. I can remove it. The reason it exists is because I started off with one of the modified HER2+ pathways which was on sharepoint not on the repo (because it wasn't being used yet). To distinguish between the original version and the new version I added the version tag.